### PR TITLE
Add basic metrics support

### PR DESCRIPTION
**Pull request by rafaelcaricio**
Send some interesting metrics to KairosDB in case the configuration is provided.

### CODE REVIEW

File: lizzy/metrics.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+class MeasureRunningTime:
+    def __init__(self, key):
+        self.key = key
+        self.start_time = datetime.datetime.now()
+
+    def finish(self):
+        end_time = datetime.datetime.now()
+        running_time = int((end_time - self.start_time).total_seconds())
+        push(self.key, running_time)
```
Comment (last-ent) [inline lizzy/metrics.py]: Wouldn't it be better to include `lizzy` as part of the key? 
As of now, that is how most of our applications are configured.

Comment (rafaelcaricio) [inline lizzy/metrics.py]: Right, I had that in mind and somehow I forgot. I will add..

Comment (last-ent) [inline lizzy/metrics.py]: Generally it is of following format:
`bus.deployctl.update.projects_created` I forgot to mention to include the team name as well.

File: lizzy/api.py [hunk 2]
```diff
@@ -4,14 +4,15 @@
...
 from decorator import decorator
 
 import connexion
-import yaml
 from flask import Response
-from lizzy import config
+from lizzy import config, metrics
 from lizzy.apps.senza import Senza
 from lizzy.exceptions import ExecutionError, ObjectNotFound, TrafficNotUpdated
+from lizzy.metrics import MeasureRunningTime
```
Comment (jmcs) [inline lizzy/api.py]: It looks a bit weird importing `metrics` from `lizzy` and then importing something from `metrics` here, but it actually reads nicer in the rest of the code.

Comment (rafaelcaricio) [inline lizzy/api.py]: So you suggest to use `metrics.MeasureRunningTime("..")`? I also found it easier to read without the prefix `metrics.*`. Or I can change the name from `MeasureRunningTime` to `RunningTime` and then use `metrics.RunningTime("..")` which seems also cool.

File: lizzy/metrics.py (hunk 1 above)
Comment (rafaelcaricio) [inline lizzy/metrics.py]: Added configuration for that.

File: lizzy/metrics.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+def push(key, value):
+    if config.kairosdb_url:
+        global server
+        if not server:
+            server = MetricWriter(kairosdb_url=config.kairosdb_url,
+                                  fail_silently=True,
+                                  timeout=4)
+        try:
+            server.write_metric(
+                '{}.lizzy.{}'.format(config.metrics_prefix, key),
```
Comment (last-ent) [inline lizzy/metrics.py]: The general format is `bus.deployctl.update.projects_created`

Basically `<team>.<project-name>.<key>` (in small letters). It just keeps them consistent with rest of the kairosdb keys.

Comment (rafaelcaricio) [inline lizzy/metrics.py]: You can configure it to whatever you want.

Comment (rafaelcaricio) [inline lizzy/metrics.py]: export METRICS_PREFIX=mybeautifulteamnamehere

Comment (last-ent) [inline lizzy/metrics.py]: 👍 makes sense.

**Comment by last-ent** [comment]
👍

**Comment by rafaelcaricio** [comment]
👍

**Pull request merged**